src/main/webapp/app/course/manage/course-group.component.ts

```diff
@@ -268,4 +274,34 @@ export class CourseGroupComponent implements OnInit, OnDestroy {
         this.rowClass = className;
         setTimeout(() => (this.rowClass = undefined));
     };
+    /**
+     * Method for exporting the csv with the needed data
+     */
+    exportStudentInformation() {
+        if (this.exportReady && this.allCourseGroupUsers.length > 0) {
+            const rows: any[] = this.allCourseGroupUsers.map((user: User) => {
+                const data = {};
+                data[NAME_KEY] = user.name!.trim();
+                data[USERNAME_KEY] = user.login!.trim();
+                data[EMAIL_KEY] = user.email!.trim();
+                return data;
+            });
+            const keys = [NAME_KEY, USERNAME_KEY, EMAIL_KEY];
```

Comment:
I think defining a type like
```ts
type CsvRow = {
    Name: string,
    Username: string,
    Email: string,
};
```
instead and using `const keys = Object.keys(rows[0]);` would work nicely here as well.